docs/design/integrate-with-cc-tools/README.md

```diff
@@ -0,0 +1,308 @@
+ //*Wrap the chaincode with FPC wrapper*//
+ cc = fpc.NewPrivateChaincode(new(CCDemo))
+ } else {
+ cc = new(CCDemo)
+ }
+ server := &shim.ChaincodeServer{
+ CCID:     ccid,
+ Address:  address,
+ CC:       cc,
+ TLSProps: *tlsProps,
+ }
+ return server.Start()
+```
+
+Note: For this code to work, there are more changes required to be done in terms of packages, building, and the deployment process. We'll go into this in the [User Experience](#user-experience) section
+
+<br>
+
+### THE CHAINCODE DEPLOYMENT PROCESS
+
+The chaincode deployment process must follow the FPC deployment flow, as CC Tools adheres to the standard Fabric chaincode deployment process, which does not account for FPC’s privacy enhancements. Since FPC is already built on top of Fabric’s standard processes, ensuring that the deployment follows the FPC-specific flow will be sufficient for integrating the two tools. This means taking care to use FPC’s specialized lifecycle commands and processes during deployment, to ensure both privacy and compatibility with the broader Hyperledger Fabric framework.
+
+1. After the chaincode is agreed upon and built (specifying the MRENCLAVE as the chaincode version), we start to initialize this chaincode enclave on the peers (Step 1).
+2. The administrator of the peer hosting the FPC Chaincode Enclave initializes the enclave by executing the `initEnclave` admin command (Step 2).
+3. The FPC Client SDK then issues an `initEnclave` query to the FPC Shim, which initializes the enclave with chaincode parameters, generates key pairs, and returns the credentials, protected by an attestation protocol (Steps 3-7).
+4. The FPC Client SDK converts the attestation into publicly verifiable evidence through the Intel Attestation Service (IAS), using the hardware manufacturer's root CA certificate as the root of trust (Steps 8-9).
+5. A `registerEnclave` transaction is invoked to validate and register the enclave credentials on the ledger, ensuring they match the chaincode definition, and making them available to channel members (Steps 10-12).
+
+![fpcFlow](./fpcFlow.png)
+
+For more details about this, refer to the FPC chaincode deployment process section in the FPC RFC document [here](https://github.com/hyperledger/fabric-rfcs/blob/main/text/0000-fabric-private-chaincode-1.0.md#deployment-process).
+
+### ON THE CLIENT-SIDE LEVEL
+
+Similarly, as cc-tools is a framework for developing the chaincode which is normally translated to fabric chaincode (now also fabric private chaincode), the client transaction invocation flow is not affected by cc-tools but it must follow the FPC transaction flow for encryption and decryption operations to meet the security requirements asserted by FPC.
+
+1. **Step 1-5: FPC Client SDK Invocation** : The client application invokes FPC Chaincode via the SDK, which retrieves the chaincode's public encryption key from the Enclave Registry and encrypts the transaction proposal in addition to a symmetric client response key.
```

Comment:
Any reason why these steps are bold face? I would remove it

Reply:
There's no reason, just for focus. 
I have removed them.